packages/opentelemetry-core/src/utils/url.ts

```diff
@@ -20,6 +20,26 @@ export function urlMatches(url: string, urlToMatch: string | RegExp): boolean {
     return !!url.match(urlToMatch);
   }
 }
+/**
+ * Check if {@param url} should be allowed when comparing against {@param allowedUrls}
+ * @param url
+ * @param allowedUrls
+ */
+export function isUrlAllowed(
```

Comment:
nit / fyi: as your introducing a new function to core, technically the xhr and fetch WILL require that they depend on this new version being released before they can consume this new function.

Generally, all versions are bumped together, but it might be worth calling out (somewhere)

Reply:
Called out the new function in core in the main changelog, but I'm not 100% sure if it's the right spot for the call out